src/server/db_slice.cc

```diff
@@ -334,29 +337,9 @@ void DbSlice::AutoUpdater::Run() {
     return;
   }
 
-  // Check that AutoUpdater does not run after a key was removed.
-  // If this CHECK() failed for you, it probably means that you deleted a key while having an auto
-  // updater in scope. You'll probably want to call Run() (or Cancel() - but be careful).
-  DCHECK(IsValid(fields_.db_slice->db_arr_[fields_.db_ind]->prime.Find(fields_.key)))
```

Comment:
why did you remove this check?
can we run now autoupdater after the key was removed?

Reply:
Nope, I should not have removed it.